pkg/asset/cluster/tfvars.go

```diff
@@ -338,16 +339,29 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		}
 		preexistingnetwork := installConfig.Config.GCP.Network != ""
 
-		imageRaw, err := rhcospkg.GCPRaw(ctx, installConfig.Config.ControlPlane.Architecture)
+		archName := coreosarch.RpmArch(fmt.Sprintf("%s", installConfig.Config.ControlPlane.Architecture))
+		st, err := rhcospkg.FetchCoreOSBuild(ctx)
 		if err != nil {
-			return errors.Wrap(err, "failed to find Raw GCP image URL")
+			return err
+		}
+		starch, err := st.GetArchitecture(archName)
```

Comment:
Can we change the name of the `starch` variable so that it is not a real word that has nothing to do with what the variable represents?
```suggestion
		stArch, err := st.GetArchitecture(archName)
```
or
```suggestion
		streamArch, err := st.GetArchitecture(archName)
```

Reply:
Hah, honestly I hadn't even noticed it was a real world and I definitely speak the English.